src/ngraph/ngraph_graph.h

```diff
@@ -219,6 +219,7 @@ class Graph : public Node {
       ngraph_forward[i] = nullptr;
       ngraph_backward[i] = nullptr;
     }
+    fprop_cache = nullptr;
```

Comment:
Just FYI in case you didn't know: the default c'tor for `shared_ptr` will initialize to nullptr.

So the `= nullptr` above isn't technically necessary.

Reply:
This is deleting it :)

Reply:
D'oh, that's what I get for reviewing a PR while listening to Jason's presentation.  Sorry :/